aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -256,10 +255,10 @@ module AptosFramework::Stake {
     ///
     /// If the stake pool is not yet active, the voting power will be 0.
     public fun get_active_staked_balance(pool_address: address): u64 acquires StakePool, ValidatorSet {
-        if (is_validator_active(pool_address)) {
-            Coin::value<TestCoin>(&borrow_global<StakePool>(pool_address).active)
-        } else {
+        if (is_validator_inactive(pool_address)) {
```

Comment:
I don't know what get_active_staked_balance is used for, but this doesn't feel right. this feels like you want is_current_validator? (include pending_inactive validator but not pending_active validator)

Reply:
This is used for on-chain governance, which checks that the lockup is longer than the voting duration. In most cases, if the voting duration is longer than epoch duration, it most likely wouldn't matter whether a validator is active, pending_active, or pending_inactive if they have a sufficient lockup as they can freely join/leave the validator set. However, the edge case here would be if the voting duration is shorter than the epoch duration. I think perhaps for simplicity, we should only count active validators?

Reply:
The nuance here is whether we only allow active validators to participate in voting or also pending_active and pending_inactive validators, as long as they have a sufficient lockup.